org.jacoco.build/licenses/asm-9.9.html

```diff
@@ -1,7 +1,7 @@
 <h4>ASM</h4>
 
 <p>
-  <a href="https://asm.ow2.io/">ASM 9.8</a> is subject to the terms and
+  <a href="https://asm.ow2.io/">ASM 9.9</a> is subject to the terms and
```

Comment:
I did check that their license stayed unchanged.

Reply:
Yes, I did check the license file too

https://asm.ow2.io/license.html

I was a bit surprised that `ASM is released under the following 3-Clause BSD License:` was left out of the copy:

<img width="719" height="733" alt="image" src="https://github.com/user-attachments/assets/6e95492f-c940-4aa2-ad9b-b99f1381a3fb" />